packages/@ember/-internals/glimmer/lib/helpers/if-unless.ts

```diff
@@ -3,40 +3,10 @@
 */
 
 import { assert } from '@ember/debug';
-import { CapturedArguments, VMArguments } from '@glimmer/interfaces';
-import { HelperRootReference } from '@glimmer/reference';
+import { VMArguments } from '@glimmer/interfaces';
+import { createComputeRef, valueForRef } from '@glimmer/reference';
 import toBool from '../utils/to-bool';
```

Comment:
semi unrelated question, why do we import `toBool` from here? I would expect for us to _always_ use `import { toBool } from '@glimmer/global-context'`

Reply:
👍 converted to using that instead